package-lock.json

```diff
@@ -2899,15 +2899,24 @@
       }
     },
     "gaxios": {
-      "version": "2.0.1",
-      "resolved": "https://registry.npmjs.org/gaxios/-/gaxios-2.0.1.tgz",
-      "integrity": "sha512-c1NXovTxkgRJTIgB2FrFmOFg4YIV6N/bAa4f/FZ4jIw13Ql9ya/82x69CswvotJhbV3DiGnlTZwoq2NVXk2Irg==",
+      "version": "2.1.0",
```

Comment:
Think these changes should be taken out since they're unrelated to the translation and likely to get into merge conflicts

Reply:
NVM, merging with latest removed it since it was done in a separate PR

Reply:
Thanks, I get it!